docs/resources/Audit_Log.md

```diff
@@ -130,25 +130,27 @@ If no object is noted, there won't be a `changes` array in the entry, though oth
 | AUTO_MODERATION_USER_COMMUNICATION_DISABLED | 145   | Member was timed out by Auto Moderation                   |                                                                                                                                                  |
 | CREATOR_MONETIZATION_REQUEST_CREATED        | 150   | Creator monetization request was created                  |                                                                                                                                                  |
 | CREATOR_MONETIZATION_TERMS_ACCEPTED         | 151   | Creator monetization terms were accepted                  |                                                                                                                                                  |
+| SET_VOICE_CHANNEL_STATUS                    | 192   | A voice channel status was changed by a user              | [Channel](#DOCS_RESOURCES_CHANNEL/channel-object)                                                                                                |
```

Comment:
there's also event `193` for when a status is removed (shows up in audit log, but for reference: [api spec](https://github.com/discord/discord-api-spec/blob/main/specs/openapi.json#L12107-L12110))